deployability/modules/workflow_engine/examples/manager/vagrant/dtt1-managers-poc-vagrant.yaml

```
@@ -0,0 +1,82 @@
+version: 0.1
+description: This workflow is used to test agents deployment por DDT1 PoC
```

Sorry, missed this one
```suggestion
description: This workflow is used to test agents' deployment for DDT1 PoC
```

---

Done in [4db5df4](https://github.com/wazuh/wazuh-qa/pull/5190/commits/4db5df48f2fffb02a146a16818f4029d890ee646)